pinot-spi/src/main/java/org/apache/pinot/spi/config/table/IndexingConfig.java

```diff
@@ -55,6 +55,7 @@ public class IndexingConfig extends BaseJsonConfig {
   private boolean _nullHandlingEnabled;
   private boolean _continueOnError;
   private boolean _validateTimeValue;
+  private boolean _skipTimeValueCheck;
```

Comment:
It is kind of confusing to have both `validateTimeValue` and `skipTimeValueCheck`, where they are not for the same purpose:
validateTimeValue: per record time value check
skipTimeValueCheck: per segment time range check

We should rename them so that it is more clear. Also we should move `continueOnError`, `validateTimeValue` and `skipTimeValueCheck` into the `IngestionConfig`